packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -368,4 +374,29 @@ export default class PciStoragesContainersService {
 
     return this.$q.resolve();
   }
+
+  getPriceEstimation(ovhSubsidiary) {
+    return this.OvhApiOrderCatalogPublic.v6()
+      .get({
+        productName: PUBLIC_CLOUD_PRODUCT_NAME,
+        ovhSubsidiary,
+      })
+      .$promise.then((catalog) => {
+        let priceObj = null;
+        find(get(catalog, 'addons', []), (addon) => {
```

Comment:
Can we use `reduce` instead, to find `priceObj`? We are not storing the result of `find` anywhere :)

Reply:
I don't think, I can use reduce here. What I am doing here is filtring array based on two nested properties and at the same time return the nested object back to avoid one more iteration.